scripts/pvscheck.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
> if it is really supported everywhere (where relevant), this could be ignored with shellcheck then?!

That's been the assumption so far in our scripts.  If we run into a problem, we can address it then.